src/app/content/highlights/components/SummaryPopup/utils.ts

```diff
@@ -1,12 +1,25 @@
 import { Highlight } from '@openstax/highlighter/dist/api';
 import queryString from 'querystring';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { persistentQueryParameters } from '../../../../navigation/selectors';
 import { Book } from '../../../types';
 import { findArchiveTreeNodeById } from '../../../utils/archiveTreeUtils';
 import { getBookPageUrlAndParams } from '../../../utils/urlUtils';
 
-export const createHighlightLink = ({ sourceId, id, anchor: elementId }: Highlight, book?: Book) => {
-  const page = book ? findArchiveTreeNodeById(book.tree, sourceId) : undefined;
-  const data = { target: JSON.stringify({ id, type: 'highlight' }) };
-  const target = queryString.stringify(data) + `#${elementId}`;
-  return `${page && book ? getBookPageUrlAndParams(book, page).url : ''}?${target}`;
+export const useCreateHighlightLink = ({ sourceId, id, anchor: elementId }: Highlight, book?: Book) => {
+  const [link, setLink] = React.useState('');
+  const persistentQueryParams = useSelector(persistentQueryParameters);
```

Comment:
im making things up a little bit here but i think when opening in a new tab you probably don't want search to be pulled with you. but you do want the content styles and archive though. unless i guess you're clicking a search result, then you'd want the search.

maybe separate your selectors into `systemQueryParameters` and rely on the router to add the `persistentQueryParameters` as it processes the navigation, that way opening a link in a new tab vs navigating through js will work right